docs/configuration.asciidoc

```diff
@@ -73,7 +73,20 @@ The URL to where the APM Server is deployed.
 
 By default the agent will validate the TLS/SSL certificate of the APM Server if using HTTPS.
 You can switch this behavior off by setting this option to `false`.
-Disabling validation is normally required if using self-signed certificates.
+
+[[server-ca-cert]]
+==== `serverCaCert`
+
+* *Type:* String
+* *Env:* `ELASTIC_APM_SERVER_CA_CERT`
+
+By default the agent will validate the TLS/SSL certificate of the APM Server using the
+well-known CAs curated by Mozilla, as described at https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options.
```

Comment:
I would prefer if the full URL wasn't displayed in the docs, but instead something like:

```suggestion
well-known CAs curated by Mozilla, as described in the Node.js docs for https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options[`tls.createSecureContext()`].
```